### PR TITLE
feat: impl patch with alpha plugin flag.

### PR DESCRIPTION
**Pull request by Peefy**
## Purpose

feat: impl patch with alpha plugin flag. More information is here: https://github.com/helmfile/helmfile/discussions/827

I found that the helmfile HelmState structure passed `chartifyOpts.EnableKustomizeAlphaPlugins=true` to `chartify`, but it did not take effect in `chartify`.

### Tests

For transformer and alpha plugin, I have added a integration test using the `AnnotationsTransformer` kustomize plugin

+ integration_test.go

### CODE REVIEW

**Comment by Peefy** [comment]
Thank you @yxxhero PTAL.

File: kustomize.go [hunk 1]
```diff
@@ -179,3 +170,36 @@ func (r *Runner) KustomizeBuild(srcDir string, tempDir string, opts ...Kustomize
...
+func (r *Runner) kustomizeEnableAlphaPluginsFlag() (string, error) {
+	version, err := r.kustomizeVersion()
+	if err != nil {
+		return "", err
+	}
+	if version.Major() > 3 {
+		return "--enable-alpha-plugins", nil
+	} else {
+		return "--enable_alpha_plugins", nil
+	}
```
[the remark this answers is not in the document]
Comment (Peefy) [reply kustomize.go]: > 

Good catch, fixed.

**Comment by yxxhero** [comment]
@Peefy please fix the ci error. then I will merge this PR.

**Comment by yxxhero** [comment]
@Peefy please see https://github.com/helmfile/chartify/pull/59/files#diff-a996645aeb193b0b8cf4f2ba786a4b4fefabf9d09b71a5382fb17681950b6d9aL161

**Comment by Peefy** [comment]
> @Peefy please fix the ci error. then I will merge this PR.

@yxxhero Sorry, I forgot to submit the `want` file and I've added it.

**Comment by yxxhero** [comment]
@Peefy please see https://github.com/helmfile/chartify/pull/59/files#diff-a996645aeb193b0b8cf4f2ba786a4b4fefabf9d09b71a5382fb17681950b6d9aL161

**Comment by Peefy** [comment]
> @Peefy please see https://github.com/helmfile/chartify/pull/59/files#diff-a996645aeb193b0b8cf4f2ba786a4b4fefabf9d09b71a5382fb17681950b6d9aL161

fixed.

**Pull request merged**